core/templates/dev/head/components/search_bar_directive.html

```diff
@@ -8,6 +8,7 @@
                 ng-class="{'glyphicon-search': !searchIsLoading,
                            'glyphicon-refresh oppia-animate-spin': searchIsLoading}">
           </span>
+          <i class="material-icons">&#xE8B6;</i>
```

Comment:
I'm not sure this is done correctly. The search bar in the gallery looks broken, and has two magnifying glasses. You probably want something like:

```
    <i class="material-icons" ng-if="searchIsLoading">...</i>
    <i class="material-icons oppia-animate-spin" ng-if="!searchIsLoading">...</i>
```

Also, $scope.searchIsLoading doesn't seem to be doing anything. I'll file a bug for that.